source/CoordinateConversion/CoordinateConversionLibrary.Tests/CoordinateConversionLibraryTests.cs

```diff
@@ -428,6 +435,109 @@ public void ParseDDM()
             Assert.IsFalse(CoordinateDDM.TryParse("This is not a coordinate", out coord));
         }
 
+        private class TestItem
+        {
+            public string Output { get; set; }
+            public int TestNumber { get; set; }
+            public double X { get; set; }
+            public double Y { get; set; }
+        }
+        private IEnumerable<TestItem> GetTestCoordinates(string jsonFile)
+        {
+            var testList = Enumerable.Empty<TestItem>();
+            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "json", jsonFile)))
+            {
+                var json = File.ReadAllText(Path.Combine(Directory.GetCurrentDirectory(), "json", jsonFile));
+                var testArray = JObject.Parse(json);
+                testList = from p in testArray["tests"]
+                               select new TestItem
+                               {
+                                   Output = (string)p["OUTPUT"],
+                                   TestNumber = (int)p["testNumber"],
+                                   X = (double)p["testString"]["x"],
+                                   Y = (double)p["testString"]["y"]
+                               };
+            }
+            return testList;
+        }
+
+        [TestMethod]
+        public void ParseDMSFromJSON()

```

Comment:
It might be helpful to move these json tests into their own test class (simple cases test class vs. automated/file based test classes)